pulumi/integration_tests/__main__.py

```diff
@@ -53,14 +53,14 @@ def _integration_container_images(
 
 def main() -> None:
     ##### Preamble
-    stack_name = pulumi.get_stack()
+    stack_name = config.STACK_NAME
 
     pulumi_config = pulumi.Config()
     artifacts = ArtifactGetter.from_config(pulumi_config)
 
     # These tags will be added to all provisioned infrastructure
     # objects.
-    register_auto_tags({"grapl deployment": stack_name})
+    register_auto_tags({"grapl_stack_name": stack_name})
```

Comment:
If we want our tags to be consistent with all our our other deployments, this should be:
```python
register_auto_tags(
    {"pulumi:project": pulumi.get_project(), "pulumi:stack": pulumi.get_stack()}
)
```